src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) throws IOException {
+        Files.write(Paths.get(filePath), data.getBytes());
+    }
```

Comment:
The class `FileWriterImpl` should be in a package that reflects its purpose. According to the checklist, it should be placed in a `service` package or a `service.impl` package if it's an implementation of a service interface.